apps/labs/posts/automated-accessibility-testing-with-bokeh.md

```diff
@@ -0,0 +1,333 @@
+and asking the larger Bokeh community whether they felt adding axe-core was
+worthwhile, or whether fixing the identified issues was a good enough action.
+
+All of this was happening while I was simultaneously facing some challenges.
+
+## Challenges
+
+The first challenge I faced was internal (within myself). I had a problem with
+not producing as much code. I have always equated productivity with having lots
+of lines of code and PRs waiting to be reviewed. At some point, I thought I
+wasn't doing enough. The idea of slow research was new to me and it took some
+getting used to, but thankfully I got around to doing more studying than coding
+for the purpose of this internship. It also paid off because I read so many
+things and got new knowledge in my repertoire. This leads me to the bigger
+challenge.
+
+Owing to the custom nature of all things Bokeh, I had a problem with getting the
+environment to allow axe-core to run. Axe-core required core modules from Node,
+and Bokeh had a stripped away version of node for the most part. I tend to like
+solving problems by myself with minimal hand-holding, but this was one of those
+scenarios that I just couldn't seem to fix without asking for help. The ways
+around the problem looked even more problematic, and I found myself questioning
+my abilities. For this problem, experience did make all the difference and my
+mentor helped me with a part of the codebase we could draw inspiration from. In
+one of our pair programming sessions, he said for us to look more closely at all
+the directories within the Javascript test directory.
+
+We found out that I had been looking at the test setup the wrong way. The bokeh
+test folder has the following [sub
+folders](https://docs.bokeh.org/en/latest/docs/dev_guide/testing.html#select-specific-bokehjs-tests):
+`baselines`, `codebase`, `defaults`, `devtools`, `integration`, and `unit`. The
+`unit`, `defaults`, and `integration` categories of tests were designed to be
+able to both run in the terminal, and on the devtools server. The `unit` and
+`integration` tests looked the most like tests that one would normally see
+and/or write in different projects because they had `describe` blocks and
+`assert` statements. Basing this assumption on just those two facts, I felt the
+location for `playwright` tests would have to model either of those two, to run
+on both terminal & devtools (which is Bokeh's server for running tests and
+viewing output in a browser), as well as with assertions. It was on closer
+inspection with my mentor in a pair programming session that we discovered that
+the `codebase` sub-directory used core node modules, and we decided to base
+playwright off that, but followed the pattern of written tests from `unit` and
+`integration` that would let the assertions and descriptions work as planned.
+This last part had to be done because Bokeh has its own custom-built test
+tooling, similar in syntax to [Jest](https://jestjs.io/), but implemented
```

Comment:
I wonder if there is some bit of confusion here.

In your [Playwright PR](https://github.com/bokeh/bokeh/pull/14032/files), the `test()` and `expect()` calls are all handled by Playwright, not by BokehJS.  So in fact, we are **not** integrating with Bokeh's internal runner here, but rather side-stepping it completely. 

What you integrated in that PR is the linting and type checking.